lib/algos/list/delimiterMaster.js

```diff
@@ -62,6 +63,11 @@ class DelimiterMaster extends Delimiter {
         let key = obj.key;
         const value = obj.value;
 
+        if (key === this.prefix) {
+            // key is of the form prefix/
+            return this.addContents(key, value);
```

Comment:
Please also test with a prefix that does not end with a delimiter character, since it should be allowed by the spec. The comment may have to be rephrased or removed then.